kuma/static/js/components/payments/payments-handler.js

```diff
@@ -434,106 +434,23 @@
             popoverBanner.removeClass('is-expanding');
             popoverBanner.attr('aria-expanded', true);
 
-            // store the current activeElement
-            activeElement = document.activeElement;
             // move focus to the secondary header text
             secondaryHeader.focus();
 
             // remove the event listener
             popoverBanner.off('transitionend');
-
-            // listen to minimise button clicks.
-            collapseButton.click(collapseCta);
-
-            $(doc).on('keydown.popoverCloseHandler', function(event) {
-                if (event.keyCode === 27) { // Escape key.
-                    collapseCta();
-                }
-            });
         });
 
         currrentPaymentForm === 'recurring'
             ? triggerRecurringPaymentEvent({
-                action: 'banner expanded',
+                action: 'banner expanded'
             })
             : triggerOneTimePaymentEvent({
                 action: 'banner',
-                label: 'expand',
+                label: 'expand'
             });
     }
 
-    /**
-     * Collapses popover.
-     */
-    function collapseCta() {
-        collapseButton.off();
-
-        // Remove error if it exists
-        if (formButton.hasClass('disabled')) {
-            toggleScriptError();
-        }
-
-        // Add transitional class for opacity animation.
-        popoverBanner.addClass('is-collapsing');
-        popoverBanner.removeClass('expanded');
-        popoverBanner.attr('aria-expanded', false);
-
-        popoverBanner.on('transitionend', function() {
-            popoverBanner.addClass('is-collapsed');
-            popoverBanner.removeClass('is-collapsing');
-            // remove the event listener
-            popoverBanner.off('transitionend');
-            // move focus back to the previous activeElement
-            activeElement.focus();
-        });
-
-        // Send GA Event.
-        triggerOneTimePaymentEvent({
-            action: 'banner',
-            label: 'collapse',
-        });
```

Comment:
Is Kadir okay losing this analytics data? And also the one below for the close event?